examples/replications-in-config.json

```diff
@@ -40,6 +52,13 @@
           "admin_channels": ["*"]
         }
       },
+      "replications": {
+        "local-continuous-false": {
+          "direction": "pull",
```

Comment:
I think it would make sense to have one of the examples be pushAndPull, so that an example of the syntax used is available.

An example with a custom conflict resolver would also be valuable.

Reply:
Cool. Added custom conflict function and also added a local wins resolution. Also added an example with all the other fields like filters and delta sync